jac-uk/admin
# #2064 Get Test Runner Working Using Vite

## What's included?
Fixed tests so that at least one runs with vitest.
Reverted changes in the following PR so they tests run on manual deploy and merge: https://github.com/jac-uk/admin/pull/2061

Closes #2064 

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?

- Run > npm test (ensuring no tests fail and at least one passes)
- Ensure the tests run on merge and manual deploy

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
